apis/user-java/helm/values.yaml

```diff
@@ -21,17 +21,17 @@ ingress:
     endpoint:
       host: changeme
       paths:
-      - path: /api/user-profile
-        serviceName: api-user-profile-svc
+      - path: /api/user-java
+        serviceName: api-user-java-svc
         servicePort: 80
-      - path: /api/healthcheck/user-profile
-        serviceName: api-user-profile-svc
+      - path: /api/healthcheck/user-java
+        serviceName: api-user-java-svc
         servicePort: 80
-      - path: /api/docs/user-profile
-        serviceName: api-user-profile-svc
+      - path: /api/docs/user-java
+        serviceName: api-user-java-svc
         servicePort: 80
-      - path: /swagger/docs/user-profile
-        serviceName: api-user-profile-svc
+      - path: /swagger/docs/user-java
```

Comment:
Please just remove this route.  We don't need it and it doesn't work in your controller anyways.